deepmd/pd/model/network/layernorm.py

```
@@ -0,0 +1,163 @@
+        ----------
+        data : dict
+            The dict to deserialize from.
+        """
+        nl = DPLayerNorm.deserialize(data)
+        obj = cls(
+            nl["matrix"].shape[0],
+            eps=nl["eps"],
+            trainable=nl["trainable"],
+            precision=nl["precision"],
+        )
+        prec = PRECISION_DICT[obj.precision]
```

Ruff

Local variable `prec` is assigned to but never used

Remove assignment to unused variable `prec`

(F841)